ceph/ceph_admin/dashboard.py

```diff
@@ -112,3 +133,121 @@ def validate_enable_dashboard(cls, username, password):
         LOG.info(
             "Dashboard API login is successful, status code : %s" % resp.status_code
         )
+
+
+def enable_alertmanager(cls, config):
+    """Method to enable alertmanager.
+    Args:
+        cls (CephAdmin object) : cephadm instance object.
+        config (Dict): Key/value pairs passed from the test suite.
+    """
+    port = config.get("alertmanager-port")
+    ssl = config.get("alertmanager_ssl_set")
+
+    # check if alertmanager daemon exist
+    out, _ = cls.shell(
+        args=["ceph", "orch", "ps", "--daemon_type", "alertmanager", "-f", "json"]
+    )
+    daemon_obj = loads(out)
+
+    if daemon_obj:
+
+        # To get hostname in which daemon got deployed
+        for daemon in daemon_obj:
+            host = daemon["hostname"]
+
+        # To get ip address of that host
+        for node in cls.cluster.get_nodes():
+            if node.hostname == host:
+                node_ip = node.ip_address
+                break
+
+        ALERTMANAGER_ENABLE_COMMANDS = [
+            f"ceph dashboard set-alertmanager-api-host 'http://{node_ip}:{port}'",
+            f"ceph dashboard set-alertmanager-api-ssl-verify {ssl}",
+        ]
+
+        execute_commands(cls, ALERTMANAGER_ENABLE_COMMANDS)
+
+        url, _ = cls.shell(args=["ceph ", "dashboard ", "get-alertmanager-api-host "])
+        validate_url(url)
+    else:
+        LOG.error("Daemon alertmanager does not exists")
+
+
+def enable_prometheus(cls, config):
```

Comment:
Could we write a separate class for `prometheus`? The idea is that these are sub commands to `dashboard` hence we could do something like
```
x = dashboard()
x.proemtheus.enable()

class Dashboard:
  def __init__(self,...):
     self.prometheus = Prometheus(...)
```

Reply:
Let's pick this up in another PR